pkg/virtctl/create/vm/vm.go

```diff
@@ -190,67 +191,119 @@ var volumeImportSizeOptional = map[string]bool{
 	ds:       true,
 }
 
+//nolint:funlen // the function is long because it has to handle many flags
 func NewCommand() *cobra.Command {
 	c := defaultCreateVM()
 	cmd := &cobra.Command{
-		Use:     "vm",
-		Short:   "Create a VirtualMachine manifest.",
-		Long:    "Create a VirtualMachine manifest.\n\nIf no boot order was specified volumes have the following fixed boot order:\nContainerdisk > PVC > DataSource > Clone PVC > Blank > Imported volumes",
+		Use:   "vm",
+		Short: "Create a VirtualMachine manifest.",
+		Long: `Create a VirtualMachine manifest.
+
+		If no boot order was specified volumes have the following fixed boot order:
+		Containerdisk > PVC > DataSource > Clone PVC > Blank > Imported volumes`,
```

Comment:
```suggestion
If no boot order was specified volumes have the following fixed boot order:
Containerdisk > PVC > DataSource > Clone PVC > Blank > Imported volumes`,
```

Otherwise output on CLI will look different. Can you please verify that?